npm/cypress-schematic/src/schematics/utils/index.ts

```diff
@@ -1,13 +1,28 @@
+import { readdirSync } from 'fs'
+import { resolve } from 'path'
+import { getSystemPath, normalize } from '@angular-devkit/core'
 import { Tree } from '@angular-devkit/schematics'
 import { get } from 'http'
 
 import { getPackageJsonDependency } from './dependencies'
+import { JSONFile } from './jsonFile'
 
 export interface NodePackage {
   name: string
   version: string
 }
 
+const ctSpecContent = ({ component, file }: {component: string, file: string}): string => {
```

Comment:
Can we use the `files/ct` template to generate the spec?

Reply:
I would really like to do that as well, but can't find a way to implement it.